browser/fxr/content/assets/icon-2d.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200"><g id="Layer_2" data-name="Layer 2"><path d="M184.9,24.29H15.1a8,8,0,0,0-8,8V167.71a8,8,0,0,0,8,8H184.9a8,8,0,0,0,8-8V32.29A8,8,0,0,0,184.9,24.29ZM23.1,159.71V40.29H176.9V159.71Z"/></g></svg>
```

Comment:
two things:
1) remove the attributes from <g> (i.e., id, data-name)
2) add the fill attribute to <svg>, i.e.
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200" fill="context-fill">`